platform/security/filter/security-filter-authorization/src/main/java/org/codice/ddf/security/filter/authorization/AuthorizationFilter.java

```diff
@@ -83,9 +83,9 @@ public void doFilter(ServletRequest request, ServletResponse response, FilterCha
 
             boolean permitted = true;
             if (shouldActionAuthorize) {
```

Comment:
With the changes that we've made to action permissions, I don't believe having this if/else even makes sense anymore. We can just do the "else" part of it always.